lib/identity_cache.rb

```diff
@@ -100,9 +101,11 @@ def should_use_cache? # :nodoc:
     # == Parameters
     # +key+ A cache key string
     #
-    def fetch(key)
+    def fetch(key, cache_fetcher_options = {})
```

Comment:
nit but I think you could splat the options hash all the way through. This way you won't have to update any assertion/expectation and everything will keep working as expected no?

Reply:
Capturing keyword arguments using `**` has the performance disadvantage of duplicating the hash for each method call.

The other annoying thing with the test expectations are that some of them use the spy gem, which doesn't yet support keyword arguments (although I've opened https://github.com/ryanong/spy/pull/12 for that issue).  This prevents me from explicitly passing options through without a splat for any method that is referenced by `Spy.on` in the test suite.

Reply:
My upstream spy PR has been merged and released, so I'll see what I can cleanup as a result of that.